v2/fleet/aws-ecr-login.timer

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Runs Amazon ECR Login every 6 hours
```

Comment:
The ECR login is only valid for 12 hours.  So I created a timer to repeat refresh the credentials.  The instances seem to reboot fairly frequently so it might not be strictly necessary.